### PR TITLE
Strip JSON spaces for compute to data.

### PR DESCRIPTION
**Pull request by calina-c**
Related to new changes in Provider. Not sure if it will pass yet, until the release of a new provider.

Fixes provider and aquarius setup being hardcoded instead of used directly from Barge.

### CODE REVIEW

**Comment by calina-c** [comment]
Blocked by Aquarius version.

**Comment by alexcos20** [comment]
we have v0.4.7, so it should be fine with that

File: .travis.yml [hunk 1]
```diff
@@ -18,9 +18,7 @@ services:
 before_install:
   - git clone https://github.com/oceanprotocol/barge
   - cd barge
-  - export PROVIDER_VERSION=ptalgs
```
Comment (trentmc) [inline .travis.yml:21]: This change doesn't seem to "strip JSON spaces", and there is no context to what it's for. 

Could you either update the PR description (and ensuing commit message) to give context, or put this in a different PR?

File: .travis.yml [hunk 2]
```diff
@@ -18,9 +18,7 @@ services:
...
   - if [[ -n "$DOCKERHUB_PASSWORD" && -n "$DOCKERHUB_USERNAME" ]]; then echo "logging in to docker";echo "$DOCKERHUB_PASSWORD" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin;fi
-  - export AQUARIUS_VERSION=v2.2.5
```
Comment (trentmc) [inline .travis.yml:23]: This change doesn't seem to "strip JSON spaces", and there is no context to what it's for. 

Could you either update the PR description (and ensuing commit message) to give context, or put this in a different PR?

**Review by trentmc — CHANGES_REQUESTED** [review]
Two of the changes don't "strip JSON spaces", and there is no context to what they are for. 

For each: please update the PR description (and ensuing commit message) to give context, or put it in a different PR.

**Comment by calina-c** [comment]
@trentmc I updated the description. This was a problem because our Travis setup was hardcoded to some previous versions of aqua and provider, instead of testing with the Barge version. These were probably introduced during tests of some features in particular, and carried along into the main branch erroneously.

"Fixes provider and aquarius setup being hardcoded instead of used from Barge."

**Review by trentmc — APPROVED** [review]

**Pull request merged**